app/src/main/java/org/seona/android/sopt/ProfileActivity.kt

```diff
@@ -0,0 +1,28 @@
+package org.seona.android.sopt
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.activity.result.ActivityResultLauncher
+import org.seona.android.sopt.databinding.ActivityLoginBinding
+import org.seona.android.sopt.databinding.ActivityProfileBinding
+import org.seona.android.sopt.databinding.ActivitySignUpBinding
+
+class ProfileActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityProfileBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityProfileBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        val name = intent.getStringExtra("name") ?: ""
+        val mbti = intent.getStringExtra("mbti") ?: ""
+
+        binding.tvProfileName.text="이름 : ${name}"
+        binding.tvProfileMbti.text="mbti : ${mbti}"
```

Comment:
```suggestion
        binding.tvProfileName.append(name)
        binding.tvProfileMbti.append(mbti)
```
`append` 사용하는걸 추천합니다! 기존 텍스트에 새로운 텍스트를 추가할 수 있어서 코드가 더 간단해지더라고요~